quickrun.el

```diff
@@ -461,7 +461,10 @@ FMT and ARGS passed `message'."
                                   (format "%%c %sKt %%a" (upcase-initials file))))))
                  (:remove  . ("%nKt.class"))
                  (:tempfile . nil)
-                 (:description . "Compile Kotlin file and execute"))))
+                 (:description . "Compile Kotlin file and execute")))
+    ("crystal" . ((:command . "crystal")
+		  (:compile-only . "%c build %s")
+		  (:description . "Run Crystal program"))))
 
```

Comment:
I think there are mix use to tabs and spaces here. Can you make sure everything goes with space instead of tab? Thanks! :) Then I think the indentation should be fixed at the same time!

Reply:
Done.